upt_macports/upt_macports.py

```diff
@@ -29,6 +31,16 @@ def _render_makefile_template(self):
         template = env.get_template(self.template)
         return template.render(pkg=self)
 
+    @property
+    def licenses(self):
+        path = 'spdx2macports.json'
+        filepath = pkg_resources.resource_filename(__name__, path)
```

Comment:
When you run "python setup.py install", is spdx2macports.json installed? Don't you need to mess with MANIFEST.in or something?